pages/challenge.js

```diff
@@ -177,10 +203,8 @@ function Challenge() {
             <h6 className={styles.centerText}>
               Here is a list of the people that have completed this before you:
             </h6>
-            <ol>
-              {challengers.map(name => (
-                <li key={name}>{name}</li>
-              ))}
+            <ol className={styles.challengerList}>
```

Comment:
You can delete this and the CSS class entirely. I don't think there will be style issues.